spec/input/_dev/spec.yml

```
@@ -17,3 +17,7 @@ spec:
       name: test
       required: false
       $ref: "./test/spec.yml"
+    - description: Folder containing shared files.
+      type: folder
+      name: shared
+      $ref: "./shared/spec.yml"
```

This could refer to the definition in the integrations directory to avoid duplicated definitions, as they are going to be the same.
```suggestion
      $ref: "../../integration/shared/spec.yml"
```